### PR TITLE
Feature/ Add option to pass custom replacement dictionaries for bias tests

### PR DESCRIPTION
**Pull request by RakshitKhajuria**
## Checklist:

- [x] I've added Google style docstrings to my code.
- [ ] I've used `pydantic` for typing when/where necessary.
- [x] I have linted my code
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: nlptest/transform/custom_bias.py [hunk 1]
```diff
@@ -2,10 +2,24 @@
     asian_names, black_names, country_economic_dict, hispanic_names, inter_racial_names, male_pronouns,
     native_american_names, neutral_pronouns, religion_wise_names, white_names, female_pronouns
 )
-from .bias import *
 
 def add_custom_data(file, name):
-    if name in CountryEconomicBias.alias_name:
+    print(type(file))
```
Comment (chakravarthik27) [inline nlptest/transform/custom_bias.py:7]: remove print statements.

**Comment by Prikshit7766** [comment]
- If `append` is `True`, the custom bias data will be appended to the existing bias data.
- If `append` is `False`, the custom bias data will replace the existing bias data.

File: nlptest/transform/custom_bias.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            - "Country Economic Bias"
+            - "Religion Bias"
+            - "Ethnicity Name Bias"
+            - "Gender Pronoun Bias"
+        append (bool): Specifies whether to append the values or overwrite them.
+
+    Raises:
+        ValueError: If the specified `name` is invalid or if the provided data has an invalid format or contains invalid keys.
+    """
+    if name == "Country Economic Bias":
```
Comment (ArshaanNazir) [inline nlptest/transform/custom_bias.py:22]: Change to 'Country-Economic-Bias'

File: nlptest/nlptest.py [hunk 3]
```diff
@@ -438,3 +439,16 @@ def load(cls, save_dir: str, model: Union[str, 'ModelFactory'], task: str,
         harness.generate()
 
         return harness
+    @staticmethod
+    def custom_bias(file_path: str, test_name: str = None, append: bool = False):
```
Comment (ArshaanNazir) [inline nlptest/nlptest.py:443]: change to 'pass_custom_bias_data'

**Pull request merged**